pynest/nest/lib/hl_api_connections.py

```diff
@@ -431,9 +410,9 @@ def CGConnect(pre, post, cg, parameter_map=None, model="static_synapse"):
 
     Parameters
     ----------
-    pre : list
+    pre : list or numpy.array
         must contain 1 subnet, or a list of GIDs
-    post : list
+    post : list or numpy.array
         must contain 1 subnet, or a list of GIDs
     cg : connection generator
         libneurosim connection generator to use
```

Comment:
This applies to the line below, where I cannot add a comment: has parameter_map the same syntax and semantics as `syn_spec` in `Connect()`? I think slightly more information would be useful.